src/NUnit3TestAdapterInstall/NUnit3TestAdapterInstall.csproj

```diff
@@ -27,27 +19,11 @@
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
     <DeployExtension>False</DeployExtension>
     <IsProductComponent>false</IsProductComponent>
```

Comment:
It seems like the eight lines above could be reduced, but I don't know enough about VSIX projects to want to try yet.